packages/calling-stateful-client/review/beta/calling-stateful-client.api.md

```diff
@@ -307,6 +308,11 @@ export interface TranscriptionCallFeature {
     isTranscriptionActive: boolean;
 }
 
+// @beta
+export interface TransferFeature {
+    acceptedTransferCallId?: string;
```

Comment:
Stateful client is designed to hold the whole state of the call at any given time. Can we store all call transfers in the stateful client?

e.g., this interface would look more like:
```ts
{
  transferRequests: {
    <callTransferId>: {
      targetParticipant: Identifier
    }
  }
}
```

and then either have a separate interface for accepted/rejected transfers:
```ts
{
  incomingTransferRequests: {
    <callTransferId>: {
      targetParticipant: Identifier
    }
  },
  acceptedTransferRequests: {
    <callTransferId>: {
      targetParticipant: Identifier
    }
  },
  rejectedTransferRequests: {
    <callTransferId>: {
      targetParticipant: Identifier
    }
  }
}
```

or a new property `status` per request:
```ts
{
  transferRequests: {
    <callTransferId>: {
      targetParticipant: Identifier,
      transferStatus: 'incoming' | 'accepted' | 'declined'
    }
  }
}
```

Reply:
cc @PorterNan

Reply:
Thanks for this comment to initiate a team discussion. I kept the transfer feature property in CallState but changed the structure to have a record of AcceptedTransfer from the team discussion.